### DISCUSSION TITLE
Railway deployment: "warn: Meilisearch error, search will be disabled"

### DISCUSSION THREAD

**Discussion by badcom** [danny-avila/LibreChat#1680] [Troubleshooting] [answered]
### What happened?

I get this error message as soon as I deploy my app to Railway. I also get a "error: [indexSync] error fetch failed" after this.
My Meilisearch variables are:

SEARCH=true
MEILI_NO_ANALYTICS=true
MEILI_HOST=http://0.0.0.0:7700
MEILI_MASTER_KEY=my_key

Am I missing something?

### Steps to Reproduce

1. Deploy the app to railway.app using the project template
2. Create a custom domain
3. Add the missing env variables such as DOMAIN_CLIENT, DOMAIN_SERVER, etc.
4. Click the LibreChat service > Deployments > View Logs

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
warn: Meilisearch error, search will be disabled
```

### Screenshots

![2024-01-30_12-29-33](https://github.com/danny-avila/LibreChat/assets/783736/4f1cabf4-40b7-4969-a07b-8fe18cd20588)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
likely cannot connect to the Meilisearch server. I can't comment without taking a dive into the config when I get the chance. I did not setup Railway but maybe @Berry-13 can comment on this.

**Comment by berry-13** [comment, marked as answer]
Sorry, that's my fault. I forgot to add the `MEILI_HOST` variable in the LibreChat deployment. Please add a variable `MEILI_HOST` with the value `${{🔎 meilisearch.MEILI_HTTP_ADDR}}`

After that, make sure the .env file looks something like this:
![image](https://github.com/danny-avila/LibreChat/assets/81851188/0118ce88-9864-41e6-a35b-701d54789e8d)

**Comment by badcom** [comment]
Ah, okay. My MEILI_HTTP_ADDR variable in the meilisearch app is currently set to [::]:3331. I guess I have to change that to the actual host url.

**Comment by badcom** [comment]
Nevermind. I hard coded the meilsearch host URL onto the MEILI_HOST var and it worked.

Thank you!
